openedx/core/djangoapps/programs/models.py

```diff
@@ -20,6 +20,13 @@ class ProgramsApiConfig(ConfigurationModel):
     public_service_url = URLField(verbose_name=_("Public Service URL"))
     api_version_number = IntegerField(verbose_name=_("API Version"))
     enable_student_dashboard = BooleanField(verbose_name=_("Enable Student Dashboard Displays"))
+    cache_ttl = IntegerField(
+        verbose_name=_("Cache Time To Live"),
+        default=0,
+        help_text=_("Time is specified in seconds. To Enable Cache Value Should Be Greater Than 0.")
```

Comment:
nit: `help_text=_("Time is specified in seconds. To enable programs cache this value should be greater than 0.")`